bump: Expo 54

ecnivtwelve opened the pull request.

Gabriel29306 reviewed (approved):
LGTM

Crazzynel reviewed (approved):
LGTM

Crazzynel commented:
Il faudra par contre une review de @tryon-dev ou encore @ecnivtwelve vu qu'on touche a l'affichage de l'App. 

Même si je pense que ça va passer hein.

ecnivtwelve committed:
chore: update dependencies and devDependencies in package.json

- Bump @expo/vector-icons to ^15.0.0
- Update @react-native-community/datetimepicker to 8.4.4
- Downgrade @react-navigation/native to ^7.1.6
- Downgrade @react-navigation/native-stack to ^7.3.10
- Update @shopify/flash-list to 2.0.2
- Upgrade expo to ^54.0.0-preview.8 and related expo packages
- Upgrade react and react-dom to 19.1.0
- Upgrade react-native to 0.81.0
- Update react-native-gesture-handler to ~2.28.0
- Upgrade react-native-safe-area-context to ~5.6.0
- Upgrade react-native-screens to ^4.15.3
- Upgrade react-native-web to ^0.21.0 and react-native-webview to 13.15.0
- Update devDependencies: @types/react to ~19.1.10, jest-expo to ~54.0.3, and typescript to ~5.9.2

ecnivtwelve committed:
Merge branch 'chokbar' of https://github.com/ecnivtwelve/chokbar into chokbar

ecnivtwelve committed:
bump: Expo 54

ecnivtwelve committed:
Merge branch 'main' into chokbar
